go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	k8s.io/apimachinery v0.24.4
 	k8s.io/autoscaler/vertical-pod-autoscaler v0.11.0
 	k8s.io/client-go v0.24.4
+	k8s.io/component-base v0.18.3
```

Comment:
looks like this is introducing an old version of component-base, while newer releases are available?